src/backend/people/settings.py

```diff
@@ -526,6 +526,14 @@ class Base(Configuration):
         environ_prefix=None,
     )
 
+    ACCOUNT_SERVICE_SCOPES = values.ListValue(
+        default=[
+            ("la-suite-list-organizations-siret", "List organizations with SIRET"),
+        ],
+        environ_name="ACCOUNT_SERVICE_SCOPES",
+        environ_prefix=None,
+    )
+
```

Comment:
Only used in the model char field choices:
 - This should be dynamic in the admin maybe
 - The default should not be our custom plugin

This can be kept like this for simplicity sake but please remove the default.